### ISSUE TITLE
combine datamaps

### ISSUE THREAD

**Issue by garfieldnate** [SoarGroup/VisualSoar#2]
It's common to combine the Soar code of multiple agents into one. In this case the datamap of any of the original agents will not suffice alone, and the entire agent can only be validated after the multiple datamaps are combined. It would be nice to automate this.

**Comment by amnuxoll** [comment]
Not *exactly* what you wanted but the new import datamap feature might be a fair substitute for this.

**Comment by garfieldnate** [comment]
How does this work? You have V-S project with a datamap and you can import multiple other project datamaps into your datamap? If so, then yes, I would say this can be closed.

**Comment by amnuxoll** [comment]
No.  You can only import the datamap and then ONLY sections that are linked directly to the top-state.  There is currently no support for importing the .soar code from another project.
